src/main/java/duke/exception/TaskStatusNotValidException.java

```diff
@@ -0,0 +1,5 @@
+package duke.exception;
+
+public class TaskStatusNotValidException extends Exception {
```

Comment:
Instead of extending directly from Exception, I think it is better to create a class for example, `DukeException` to add more abstraction layer to it.